bin/pyenv-installer

```diff
@@ -22,7 +22,7 @@ colorize() {
 }
 
 checkout() {
-  [ -d "$2" ] || git clone "$1" "$2"
+  [ -d "$2" ] || git clone --depth 1 "$1" "$2"
```

Comment:
Ah it seems I was misunderstanding. It seems we can `git pull` to fetch latest changes even with shallow clone repository. If it works fine, we can use `--depth 1` by default.